src/Fixer/Preload/PreloadExplicitClassSymbolsFixer.php

```diff
@@ -0,0 +1,282 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Preload;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Analyzer\ArgumentsAnalyzer;
+use PhpCsFixer\Tokenizer\Analyzer\FunctionsAnalyzer;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+use PhpCsFixer\Tokenizer\TokensAnalyzer;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class PreloadExplicitClassSymbolsFixer extends AbstractFixer
+{
+    private $functionsAnalyzer;
+    private $tokenAnalyzer;
+
+    public function __construct()
+    {
+        parent::__construct();
+        $this->functionsAnalyzer = new FunctionsAnalyzer();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Adds extra `class_exists` before a class to help opcache.preload to discover always-needed symbols.',
```

Comment:
Nice idea to make this as a fixer!

Please provide a `$description` parameter with the reasoning, especially:
- why the rule is worth to use?
- `what classes we don't need to preload, ie classes that is found by reflection` and why

Reply:
I added a desciption